packages/functional-tests/src/app.ts

```diff
@@ -58,14 +60,18 @@ export default class App {
         } else {
             testName = this.params.test as string;
         }
-
-        await this.startTest(testName);
+        if (testName) {

```

Comment:
Did you want to move this logic into onstarted? Seems like the more correct thing to do.